File: workspace/javascript/linearSearch.js
let sample = [2,3,4];
let target = 4;

const linearSearch = (function(arr, target) {
  for(let index = 0; index < arr.length; index++) {
    if(arr[index] == target) {
      return `Found ${target} at arr[${index}]`;
    }
  }
  return "Not Found";
})(sample, target);

console.log(linearSearch);

File: workspace/java/linearSearch.java
public class linearSearch {
    public static int linearSearchNum(int[] list, int target) {
        for (int i = 0; i < list.length; i++) {
            if (list[i] == target) {
                return i;
            }
        }
        return -1;

    }
    public static void main(String[] arg) {
        int[] arr = {1,2,3,4,5};
        int target = 5;
        System.out.println("Target " + target + " is found at index: " + linearSearchNum(arr, target));  
    }
}